.github/workflows/node.yml

```diff
@@ -11,7 +11,6 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v4
       - name: Code Review
-        uses: nishtahir/conventional-gpt@main
+        uses: nishtahir/conventional-gpt@develop
```

Comment:
Updating the action from `nishtahir/conventional-gpt@main` to `nishtahir/conventional-gpt@develop` should be approached with caution. Ensure that the `develop` branch is stable and tested, especially if this is used in a production environment.", "side": "RIGHT